src/plugins/manualColumnResize/manualColumnResize.js

```diff
@@ -337,14 +338,14 @@ class ManualColumnResize extends BasePlugin {
    * @returns {boolean}
    */
   checkIfColumnHeader(element) {
```

Comment:
This shape of the code reminds me of our `closest` helper. Locally I've replaced the whole method body with `return closest(element, ['THEAD'], this.hot.rootElement) ? true : false;`, and after run test, nothing goes red. Do you think that that part of the code can be enhanced in that way?